server/actions/inviteCodes.js

```diff
@@ -15,25 +13,30 @@ export async function findInviteCodesToExpire(since = null) {
     ))
   }
 
-  const inviteCodesNotRecentlyUsed = await r.table('users')
-    .filter(user => user('createdAt').gt(codeExpirationDate).not())('inviteCode')
-    .distinct()
+  const inviteCodesNotRecentlyUsed = [...new Set(
+    await User
+      .run()
+      .filter(user => user.createdAt <= codeExpirationDate)
+      .map(user => user.inviteCode)
```

Comment:
Nothing should be chained after `.run()`. We can use the old `.filter()` and `.distinct()` followed by `.execute()` instead of `.run()` to maintain the old functionality.